modules/test/playwright/tests/portlet-configuration-css-web/textStyle.spec.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {pageViewModePagesTest} from '../../fixtures/pageViewModePagesTest';
+import {pagesAdminPagesTest} from '../../fixtures/pagesAdminPagesTest';
+import {productMenuPageTest} from '../../fixtures/productMenuPageTest';
+import {uiElementsPageTest} from '../../fixtures/uiElementsTest';
+import getRandomString from '../../utils/getRandomString';
+
+export const test = mergeTests(
+	loginTest(),
+	pagesAdminPagesTest,
+	pageViewModePagesTest,
+	productMenuPageTest,
+	uiElementsPageTest
+);
+
+test('LPD-31641 - Look & Feel, center text alignment test for LPD-16327', async ({
+	page,
+	pagesAdminPage,
+	productMenuPage,
+	uiElementsPage,
+	widgetPagePage,
+}) => {
+	await productMenuPage.openProductMenuIfClosed();
+	await productMenuPage.goToPages();
+	await uiElementsPage.clickNewButton();
+	if (!pagesAdminPage.addPageButton.isVisible) {
```

Comment:
Is this conditional really needed? I think it shouldn't be... the previous call `uiElementsPage.clickNewButton()`; already has a waitFor() for the new button, and it should open the submenu.

```
async clickNewButton() {
	await this.newButton.waitFor({state: 'visible'});
	await this.newButton.click();
}
```

Reply:
In the end I wont need uiElementsPage at all. Thanks